src/prisma/migrations/20230821120539_auto_lens_publish/migration.sql

```diff
@@ -0,0 +1,5 @@
+-- DropIndex
+DROP INDEX "User_xpsEngineId_key";
```

Comment:
please remove htis

Reply:
Thank you for catching this. Fixed. That was a close one 😬 need to be more careful